spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -40,7 +40,7 @@
     then_I_see_the_new_step_by_step_page
   end
 
-  scenario "User visits an existing step by step page" do
+  scenario "User visits an existing step by step page with no steps" do
     given_there_is_a_step_by_step_page
```

Comment:
Should change this to `given_there_is_a_step_by_step_page_with_no_steps`, now that I've made an alias method for it.